doc/README.build_env.md

```diff
@@ -114,3 +114,35 @@ optional arguments:
                         False)
 ==============================================================================
 ```
+
+## Conda environment files
+
+`build_env.py` also generates a conda environment file based on the configuration
+ selected for build. For e.g. if `build_env.py` is run for tensorflow-env.yaml and
```

Comment:
selected for build. For e.g. if `build_env.py` is run for `tensorflow-env.yaml` and